hackathon_site/event/static/event/styles/scss/styles.scss

```diff
@@ -318,6 +325,34 @@ strong {
         display: block;
     }
 }
+
+.input-field {
+    input:not([type]):focus:not([readonly]) + label,
```

Comment:
What does all this magic do?

Reply:
Make the input fields the primary color. I tried what was mentioned on the documentation and it didnt work so I just took the class from inspect element to hack it